tests/crypto_policy_test.go

```diff
@@ -103,6 +106,19 @@ var _ = Describe("Crypto Policy", func() {
 
 	AfterEach(func() {
 		strategy.RevertToOriginalSspCr()
+
+		// Because of bug[1], the SSP operator will move to CrashLoopBackOff state,
```

Comment:
supernit - I forget what the rules are for FIXMEs in this codebase but this strikes me as something we could mark as FIXME once that bug is resolved?

Reply:
There are no rules. Do you have a suggestion for what is a good way to mark as FIXME? Or simply prefix the comment with `FIXME:` ?